edx/edx-enterprise
# ENT-101: Modify view and templates to allow for course-specific consent questioning

**Description:**

This pull request updates the data sharing consent views and templates to enable it to meet the requirements of collecting data sharing consent for a particular course.

**JIRA:** [ENT-101](https://openedx.atlassian.net/browse/ENT-101)

**Dependencies:** None

**Merge deadline:** 30 January 2017

**Testing instructions:**

See [testing instructions for edx-platform PR #14400](https://github.com/edx/edx-platform/pull/14400).

**Reviewers:**

**Merge checklist:**

**Post merge:**
      finished.

**Author concerns:** Additional tests are in progress, and I need to figure out the best strategy to prevent the form from redirecting to an arbitrary destination.
